app/page.js

```diff
@@ -244,6 +270,54 @@ export default function Home() {
         </>
       )}
       <ClientJs setFingerPrint={onSetFingerPrint} />
+
+      <Modal hideCloseButton isOpen={isOpen}
```

Comment:
@rozzu can we create a separate component (in a separate file) for this thing? this will make the code clean. What do you think?